src/quick-lint-js/fe/parse.cpp

```diff
@@ -553,51 +553,11 @@ namespace quick_lint_js
 
   void parser::warn_on_mistyped_strict_inequality_operator(source_code_span op_span)
```

Comment:
Minor: Because this function depends on non-obvious state, I think you should inline it into its call in `parse_expression_remainder`.

Reply:
You mean as in `void parse_expression_remainder::warn_on_mistyped_strict_inequality_operator(source_code_span op_span)`?

Reply:
I don't understand your question. (Copy paste mistake?)

I'm saying: you have the following:
```c++
void parse_expression_remainder() {
  ...
  warn_on_mistyped_strict_inequality_operator(x);
  ..
}

void warn_on_mistyped_strict_inequality_operator(op_span) {
  (do stuff with op_span)
}
```
but I think you should write the following instead:
```c++
void parse_expression_remainder() {
  ...
  (do stuff with x)
  ..
}

// The warn_on_mistyped_strict_inequality_operator function no longer exists.
```

Reply:
(I checked 6219bcc6499d6431d9e08a6a75557bb3b7f7f126 and it looks like you understood what I meant. :+1:)